sds_data_manager/lambda_code/IAlirtCode/ialirt_ingest.py

```diff
@@ -19,6 +27,93 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+KERNELS = {
+    "ephemeris_predicted",
+    "ephemeris_90days",
+    "planetary_ephemeris",
+    "spacecraft_clock",
+    "leapseconds",
+    "imap_frames",
+    "science_frames",
+}
+EFS_BASE_PATH = Path("/mnt/data")
+
+
+def get_ephemeris_time():
+    """Get the current ephemeris time in seconds since J2000.
+
+    Returns
+    -------
+    et: float
+        The current ephemeris time in seconds since J2000.
+    """
+    now = datetime.now(timezone.utc)
+    j2000 = datetime(2000, 1, 1, 12, 0, 0, tzinfo=timezone.utc)
+    et = (now - j2000).total_seconds()
+    return et
+
+
+def get_latest_spice_kernels() -> ProcessingInputCollection:
+    """Query the SPICE metakernel API for latest SPICE kernel filenames.
+
+    Returns
+    -------
+    dependency_inputs: ProcessingInputCollection
+        A collection containing a SPICEInput object with the list of kernel filenames
+        returned from the metakernel API.
+    """
+    dependency_inputs = ProcessingInputCollection()
+
+    now = datetime.now(timezone.utc)
+    one_week_ago = now - timedelta(weeks=1)
+    # Define J2000 epoch: 2000-01-01T12:00:00 UTC
+    j2000 = datetime(2000, 1, 1, 12, 0, 0, tzinfo=timezone.utc)
+    end_time = (now - j2000).total_seconds()
+    start_time = (one_week_ago - j2000).total_seconds()
```

Comment:
This is converting to ET. Might be worth naming these variables to reflect that.
```suggestion
    et_end_time = (now - j2000).total_seconds()
    et_start_time = (one_week_ago - j2000).total_seconds()
```

IMO, we should ask Bryan to add UTC time strings as a query option. For now, this estimated time approach is probably OK.